ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineNewPayloadV3Test.java

```diff
@@ -108,20 +105,8 @@ public void shouldInvalidVersionedHash_whenShortVersionedHash() {
 
   @Test
   public void shouldValidVersionedHash_whenListIsEmpty() {
-    final BlockHeader mockHeader =
-        setupValidPayload(
-            new BlockProcessingResult(Optional.of(new BlockProcessingOutputs(null, List.of()))),
-            Optional.empty(),
-            Optional.empty());
-    final EnginePayloadParameter payload =
-        mockEnginePayload(mockHeader, Collections.emptyList(), null, null);
-
-    ValidationResult<RpcErrorType> res =
-        method.validateParameters(
-            payload,
-            Optional.of(List.of()),
-            Optional.of("0x0000000000000000000000000000000000000000000000000000000000000000"));
-    assertThat(res.isValid()).isTrue();
+    final JsonRpcRequestContext mock = getRequestContextMock(null, 1L, "1", new String[0], "0x0");
```

Comment:
nit: would rename this mockRequestContext() so it implies it is creating something, not getting an already existing thing.